Automattic/pocket-casts-ios
# Referrals: update share message

| 📘 Part of: #2083  |  
|:---:|

Fixes # 



<img src="https://github.com/user-attachments/assets/80433556-bee8-4b1a-a811-e14e0a72a585" width=320>

## To test

1. Start the app
2. Login with a Plus or Patron account
3. Go to Profile
4. Tap on the gift icon on the top left
5. Tap on Share Guest Pass
6. Choose email
7. Check to see if you the update message and an image of the guest pass on the email content

## Checklist
